Convert 'What to do if there are teamwork issues' doc to html #47

Foleosy opened the pull request:
Fixes #47

Foleosy commented:
I was unfortunately unable to figure out how to test if the dropdown page function in week9.html works properly.

damithc commented:
To test, open index.html in IE (not chrome or firefox). If other similar embedded links are working but not the new one, then there is a bug in your code.

damithc commented on contents/handbook-appendixF.html:
```diff
@@ -0,0 +1,13 @@
+  <li><strong>Redistribute the work</strong>: Stronger programmers in the team should take over the critical parts of the code.</li>
+  <li><strong>Enforce stricter integration workflow</strong>: Appoint an integrator (typically, the strongest programmer). His/her job is to maintain the integrated version of the code. He/she should not accept any code that breaks the existing product or is not up to the acceptable quality standard. It is up to others to submit acceptable code to the integrator.
+Note that if the integrator rejected your code unreasonably, you can still earn marks for that code. You are allowed to submit such &#39;rejected&#39; code for grading. They can earn marks based on the quality of the code. Also refer the workflow described in the &#39;A safer workflow using forks and pull requests&#39; section of the Handbook: Appendix E (Using GitHub Project Hosting)</li>
+</ul>
```
Limit line length to about 110-120 chars.

damithc commented on contents/week9.html:
```diff
@@ -28,8 +28,10 @@ <h3 id="things-to-do-content-week9" class="buttoned"><span>Things to do</span></
+                <span onclick="pullContent('handbook-appendixF', '#embedded-week9-handbook', 'Handbook: Appendix F (What to do if there are teamwork issues)');">
+                <span class="embedded-link">Handbook: Appendix F (What to do if there are teamwork issues).</span>
+                </span>
+                <div id="embedded-week9-handbook"></div>
```
This div id needs to be unique. Don't reuse the same div id as the place you copied this code.

Foleosy commented:
Thank you for your advice, as well as being so understanding.

I have fixed the issues above, and tested the index.html in IE 11; both handbook and schedule pages worked fine.

Apologies for not fixing the div id after copying (without permission at that).

Foleosy commented:
....and I mis-clicked the close and comment button, sorry.

damithc commented on contents/handbook-appendixF.html:
```diff
@@ -0,0 +1,39 @@
+<ul>
+  <li>First, <span class="highlighted">do not expect everyone to have the same skill/motivation level 
+as you</span>. It is fine if someone wants to or less and have low expectations from the module. That 
+doesn&#39;t mean that person is a bad person. Everyone is entitled to have their own priorities.</li>
```
use `'` instead of `&#39;`, unless there is a strong reason for using the latter?

damithc commented on contents/handbook-appendixF.html:
```diff
@@ -0,0 +1,39 @@
+&#39;rejected&#39; code for grading. They can earn marks based on the quality of the code. Also 
+refer the workflow described in the &#39;A safer workflow using forks and pull requests&#39; 
+section of the Handbook: Appendix E (Using GitHub Project Hosting)</li>
+</ul>
```
Indentation seem to be off. Most lines are not indented at all.

damithc commented:
Added some comments.

Foleosy commented:
I originally used the escape characters for quote because the single quote wouldn't display properly, but it looks like it works fine now, so I've changed it back.

Apologies about the indentation; forgot about it when I changed the length of lines.

damithc commented:
I'll review in a couple of days. A bit busy now.
